entity/contents/filters/border_mask_blur_filter_contents.cc

```diff
@@ -108,18 +108,21 @@ bool BorderMaskBlurFilterContents::RenderFilter(
   return pass.AddCommand(std::move(cmd));
 }
 
-std::optional<Rect> BorderMaskBlurFilterContents::GetCoverage(
+std::optional<Rect> BorderMaskBlurFilterContents::GetFilterCoverage(
+    const FilterInput::Vector& inputs,
     const Entity& entity) const {
-  auto coverage = FilterContents::GetCoverage(entity);
+  if (inputs.empty()) {
+    return std::nullopt;
+  }
+
+  auto coverage = inputs[0]->GetCoverage(entity);
```

Comment:
here and on the guassian filter below: why are we only using the first input? Do we expect there to only ever be one? 

It seems confusing to me that this API sometimes takes and uses multiple vector inputs and other times only uses one.

Reply:
This funkiness is a side effect of making filter inputs agnostic to the filter implementation (partly inspired by the design of [Core Image](https://developer.apple.com/library/archive/documentation/GraphicsImaging/Conceptual/CoreImaging/ci_tasks/ci_tasks.html#//apple_ref/doc/uid/TP30001185-CH3-TPXREF101)).

This detail used to be more important because `FilterContents` would resolve all the filter inputs as textures before calling the render method, but that work is now deferred through `FilterInputs`. There's still a default coverage routine that most filters won't need to override in practice, but that's been majorly simplified now that everything has a `GetCoverage`, including `FilterInput` itself.

Creating Filters in Impeller is locked down to static factory methods in `FilterContents` that take the intended number of inputs. In the filters themselves, too many inputs isn't an error, but not enough inputs fails the render.

At this point I'm not really married to this detail, but keeping it around might make creating a debug view for filters easier. 🤷  We can probably remove it sometime since it's not really important anymore.